wtc/wtc.py

```diff
@@ -160,12 +160,13 @@ def pack_bytes(bs):
 
     return lzma.compress(buf, format=2)
 
-def decompress(compressed_lzma):
+def decompress(compressed_lzma, return_decompressed_lzma=False):
     """
     Decompresses a separated and compressed lzma into an lzma stream.
 
     Args:
         String compressed_lzma: A separated and compressed representation of replay data.
+        Boolean return_decompressed_lzma: Whether to return decompressed and decoded lzma, after decompressing from wtc.

```

Comment:
could we go with something shorter like `decompressed` ?

Reply:
Unfortunately just decompressed would be vague - there’s really two levels of compression here, wtc and lzma. We should specify which one is being decompressed. The documentation / function names should really be changed as well, but am lazy.

Reply:
then please use `decompressed_lzma`, the return is unnecessary.